cassovary-core/src/test/scala/com/twitter/cassovary/graph/bipartite/BipartiteGraphSpec.scala

```diff
@@ -13,19 +13,18 @@
  */
 package com.twitter.cassovary.graph.bipartite
 
-import com.twitter.cassovary.graph.bipartite.BipartiteGraphDir._
 import com.twitter.cassovary.graph.GraphDir
-import com.twitter.cassovary.graph.GraphDir._
-import org.specs.Specification
+import org.scalatest.WordSpec
+import org.scalatest.matchers.ShouldMatchers
 import scala.collection.mutable
 
-class BipartiteGraphSpec extends Specification {
+class BipartiteGraphSpec extends WordSpec with ShouldMatchers {
 
   var graph: BipartiteGraph = _
```

Comment:
this is not your problem, but this test has been written previously pretty horribly (use of vars, side-effects, etc.) This is a good opportunity to fix it.

Please remove all vars and side-effects.